tests/src/main/scala/tests/Suite.scala

```diff
@@ -0,0 +1,105 @@
+package tests
```

Comment:
Instead of rolling our own testing framework here, how about using [minitest](https://github.com/monix/minitest/)?